qa/rpc-tests/wallet_overwintertx.py

```diff
@@ -141,6 +141,10 @@ def run_test (self):
         self.nodes[0].generate(1)
         self.sync_all()
         bci = self.nodes[0].getblockchaininfo()
+
+        # size_on_disk should be > 0
+        assert_greater_than(bci['size_on_disk'], 0)
```

Comment:
I feel like this test makes more sense in `blockchain.py` (as in the upstream PR), but I suppose in the grand scheme this doesn't really matter. This would be easier if we had already pulled in https://github.com/bitcoin/bitcoin/pull/11370 which adds the call to `getblockchaininfo` in that test.

Reply:
> Looks good. I am curious if other people have any thoughts on my questions/comments above.

Doesn't seem we pulled in the pruning code, so not sure it would be useful.